import-export-cli/cmd/addEnv.go

```diff
@@ -99,9 +99,9 @@ func addEnv(envName string, envEndpoints *utils.EnvEndpoints, mainConfigFilePath
 	}
 
 	if envEndpoints.TokenEndpoint == "" {
-		// if mandatory token endpoint is blank
-		utils.ShowHelpCommandTip(addEnvCmdLiteral)
-		return errors.New("Token endpoint cannot be blank")
+		// If token point string is empty assign the default value
+		envEndpoints.TokenEndpoint = "https://localhost:9443/oauth2/token"
```

Comment:
Please add this as a constant and use the constant here instead of using the string itself.